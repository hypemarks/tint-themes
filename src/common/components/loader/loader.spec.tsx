import { render } from '@testing-library/react';
import React from 'react';
import Loader from './loader';

describe('Test <Loader /> Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(<Loader />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
