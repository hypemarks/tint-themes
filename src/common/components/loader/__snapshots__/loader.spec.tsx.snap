// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Loader /> Component should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="loader"
    data-testid="loader"
  >
    <img
      alt="tint"
      src="https://cdn.hypemarks.com/assets/splash5/Logo_Icon_warmRed_@2x.png"
    />
  </div>
</DocumentFragment>
`;
